services/QuillLMS/app/services/classlink_integration/user_persister.rb

```diff
@@ -83,8 +99,8 @@ def self.creatable_user(user_hash) = user_hash.except(:sourced_id, :given_name,
     def self.updatable_user(user_hash) = creatable_user(user_hash).except(:username)
 
     private def handle_error(error)
-      msg = "Failed to persist classlink user with sourcedId=#{incoming_user[:sourced_id]}"
-      ErrorNotifier.report(error, { message: msg })
+      msg = "Failed to persist classlink user with sourcedId=#{incoming_user[:sourced_id]} | #{error}"
```

Comment:
Taking your suggestion from a different PR: https://github.com/empirical-org/Empirical-Core/pull/12984/files#r2091554057

I'm wrapping the root error in a Classlink-namespaced error, but want to keep the root error message, for diagnostic purposes